lib/api/v1/routers/router.go

```diff
@@ -1,13 +1,23 @@
 package routers
 
 import (
-	"github.com/markgravity/golang-ic/lib/api/v1/controllers"
-
 	"github.com/gin-gonic/gin"
+	"github.com/markgravity/golang-ic/lib/api/v1/controllers"
+	"github.com/markgravity/golang-ic/lib/services/oauth"
 )
 
 func ComebineRoutes(engine *gin.Engine) {
 	v1 := engine.Group("/api/v1")
 
 	v1.GET("/health", controllers.HealthController{}.HealthStatus)
+
+	// TODO: Correct the logic in Sign-In task
+	v1.POST("/auth/sign-in", func(context *gin.Context) {
```

Comment:
Normally, we based on the resources (e.g., `Token`, `User`) to name the endpoint

```suggestion
	v1.POST("/oauth/token", func(context *gin.Context) {
```

Reply:
(had the same suggestion, then I looked in Varun and it used sign-in 😂 )
But I agree, better to use resource driven routes (unless there is a business requirement specific to the routing).

Reply:
@suho Please ignore this implementation, it's for POW purposes only. 
I will correct it in #25